tests/nlu/base/test_config.py

```diff
@@ -40,13 +42,18 @@ def test_invalid_pipeline_template():
     assert "unknown pipeline template" in str(execinfo.value)
 
 
-def test_pipeline_looksup_registry():
-    pipeline_template = list(registered_pipeline_templates)[0]
+@pytest.mark.parametrize(
+    "pipeline_template", list(registered_pipeline_templates.keys())
+)
+def test_pipeline_looksup_registry(pipeline_template: Text):
```

Comment:
```suggestion
def test_pipeline_looks_up_registry(pipeline_template: Text):
```

Reply:
or this one, right? 
```suggestion
def test_pipeline_lookup_registry(pipeline_template: Text):
```

Reply:
well if you mean "the lookup" it would be
```suggestion
def test_pipeline_registry_lookup(pipeline_template: Text):
```
😄